README.md

```diff
@@ -1,83 +1,88 @@
-| SSAS_MFA_TOKEN_ <br/> TIMEOUT_MINUTES | No | X |   | Token lifetime for self-registration (MFA tokens and Registration tokens).  Defaults to 60 (minutes). |
-| SSAS_READ_TIMEOUT    | No       | X |   | Sets the read timeout on server requests |
-| SSAS_WRITE_TIMEOUT   | No       | X |   | Sets the write timeout on server responses |
-| SSAS_IDLE_TIMEOUT    | No       | X |   | Sets how long the server will keep idle connections open |
-| SSAS_LOG                     | No  | X |   | Directs all ssas logging to a named file |
-| SSAS_ADMIN_PORT <br/> SSAS_PUBLIC_PORT <br/> SSAS_HTTP_TO_HTTPS_PORT | No  | X | X | These values are not yet used by code. Intended to allow changing port assignments. If used, will affect BCDA SSAS URL vars. |
-| SSAS_ADMIN_SIGNING_KEY <br/>or<br/> SSAS_ADMIN_SIGNING_KEY_PATH| Yes | X |   | Provides the plaintext/path of the admin server signing key. When setting vars for AWS envs, you must include a var for the key material. Set either the key or the path but not both. |
-| SSAS_PUBLIC_SIGNING_KEY <br/>or <br/>SSAS_PUBLIC_SIGNING_KEY_PATH | Yes | X |   | Provides the plaintext/path of the public server signing key. When setting vars for AWS envs, you must include a var for the key material. Set either the key or the path but not both. |
-| SSAS_TOKEN_BLACKLIST_CACHE_ <br/> CLEANUP_MINUTES  | No | X | | Tunes the frequency that expired entries are cleared from the token blacklist cache.  Defaults to 15 minutes. |
-| SSAS_TOKEN_BLACKLIST_CACHE_ <br/> TIMEOUT_MINUTES  | No | X | | Sets the lifetime of token blacklist cache entries.  Defaults to 24 hours. |
-| SSAS_TOKEN_BLACKLIST_CACHE_ <br/> REFRESH_MINUTES  | No | X | | Configures the number of minutes between times the token blacklist cache is refreshed from the database. |
-| BCDA_TLS_CERT        | Depends  | X |   | The cert used when the SSAS service is running in secure mode. When setting vars for AWS envs, you must include a var for the cert material. This var should be renamed to SSAS_TLS_CERT. |
-| BCDA_TLS_KEY         | Depends  | X |   | The private key used when the SSAS service is running in secure mode. When setting vars for AWS envs, you must include a var for the key material. This var should be renamed to SSAS_TLS_KEY. |
-| SSAS_CLIENT_ASSERTION_AUD         | Yes  | X |   | The audience (aud) claim value required when authenticating using client assertion tokens (v2/token). |
+| Key                                                                  | Required | SSAS | BCDA | Purpose                                                                                                                                                                                                             |
```

Comment:
The markdown table format doesn't look quite right.  

Try the following for line 36 (there was extra pipes and you need at least 3 hyphens)
`|----------------------------------------------------------------------|:--------:|:----:|:----:|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|`

Line 45 has two extra pipes at the end.